src/MinterGateway.sol

```diff
@@ -881,12 +877,13 @@ contract MinterGateway is IMinterGateway, ContinuousIndexing, ERC712Extended {
      * @param newTimestamp_ The timestamp of the collateral update.
      */
     function _updateCollateral(address minter_, uint240 amount_, uint40 newTimestamp_) internal {
-        MinterState storage minterState_ = _minterStates[minter_];
```

Comment:
We should keep this variable.

Reply:
Yes, the PR was a draft that I don'y even think compiled without this line.